akka-persistence-typed-tests/src/test/scala/akka/persistence/typed/jackson/ReplicatedEventSourcingJacksonSpec.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.typed.jackson
+
+import akka.actor.testkit.typed.scaladsl.{ LogCapturing, ScalaTestWithActorTestKit, SerializationTestKit }
+import akka.persistence.typed.ReplicaId
+import akka.persistence.typed.crdt.{ Counter, LwwTime, ORSet }
+import akka.persistence.typed.jackson.ReplicatedEventSourcingJacksonSpec.{ WithCounter, WithLwwTime, WithOrSet }
+import akka.serialization.jackson.{ AkkaSerializationDeserializer, AkkaSerializationSerializer, JsonSerializable }
+import com.fasterxml.jackson.databind.annotation.{ JsonDeserialize, JsonSerialize }
+import org.scalatest.wordspec.AnyWordSpecLike
+
+object ReplicatedEventSourcingJacksonSpec {
+  final case class WithLwwTime(lwwTime: LwwTime) extends JsonSerializable
```

Comment:
how can it work without the annotations?

Reply:
ah, it's a good enough case class to be able to serialize as native json

Reply:
Yep and the counter works without it oo but i added a test with it